scripts/src/main/java/io/github/fate_grand_automata/scripts/entrypoints/AutoServantLevel.kt

```diff
@@ -250,9 +267,43 @@ class AutoServantLevel @Inject constructor(
      * @see isAutoSelectVisible
      */
     private fun performAutoSelect() {
+        if (stateAutoFill) {
+            locations.enhancementClick.click()
```

Comment:
This one clicks on Enhance but if there's no autofill it doesn't do that.

Isn't it a better to idea to combine the `performAutoSelect` and `performEnhancement` methods?